code/src/components/app-link/AppLink.scss

```diff
@@ -0,0 +1,19 @@
+@import "@design-system";
+
+.spa-link {
+  text-decoration: none;
+  color: var(--spa-link-primary-color, $black);
+  transition: all 300ms;
```

Comment:
maybe use transition only for color and also time in format .3s?

Reply:
Fixed